test/e2e/cmd/chaos/command.go

```diff
@@ -0,0 +1,71 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package chaos
+
+import (
+	"time"
+
+	logutil "github.com/elastic/cloud-on-k8s/pkg/utils/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var (
+	log = logf.Log.WithName("chaos")
+
+	defaultDeleteOperatorPodDelay            = 9 * time.Minute
```

Comment:
Could you add a comment explaining why those values are being used (how they were picked)?

Reply:
I will do. I think a nice improvement (to be done in a separate PR) would be use something like https://github.com/lthibault/jitterbug to make the Pod deletion less "predictable" and more "chaotic"